src/nyc_trees/gulpfile.js

```diff
@@ -153,6 +153,7 @@ function browserifyTask(bundler) {
         })
         .bundle()
         .on('error', gutil.log.bind(gutil, 'Browserify Error'))
+        .on('error', process.exit.bind(process, 1))
```

Comment:
Well it's a nice to have really.

We could be a little messy and just pass in a boolean `isWatchify` parameter to `browserifyTask`, and use that plus `gulpif` to gate whether or not error should exit the process.